dataclients/kubernetes/crd.md

```diff
@@ -0,0 +1,14 @@
+# RouteGroup CRD
+
+This document is a temporary design document, continously updated as this development branch evolves.
+
+## Goal of the RouteGroup CRD
+
+Provide a format for routing input that makes it easier to benefit more from Skipper's routing features, without
+workarounds, than the generic Ingress specs. Primarily targeting the Kubernetes Ingress scenario.
```

Comment:
"..., without workarounds that are required when using the generic Ingress specs..."

Reply:
this part was rewritten in a concurrent change :)